src/ai/susi/server/api/aaa/ChangeUserRoles.java

```diff
@@ -149,15 +149,15 @@ public ServiceResponse serviceImpl(Query query, HttpServletResponse response, Au
             ClientCredential credential = new ClientCredential(ClientCredential.Type.passwd_login, userTobeUpgraded);
             ClientIdentity identity = new ClientIdentity(ClientIdentity.Type.email, credential.getName());
             if (!DAO.hasAuthorization(identity)) {
-                throw new APIException(400, "Username not found");
+                throw new APIException(404, "Username not found");
```

Comment:
It shouldn't be a 404 as the route is present and a param of username provided is only wrong